lib/rest-client.js

```diff
@@ -29,7 +29,7 @@ module.exports = class RestClient {
   }
 
   async getActiveOrder(params) {
-    return await this.request.get('open-api/order/list', params);
+    return await this.request.get('v2/private/order/list', params);
   }
```

Comment:
Good catch, I thought I covered the deprecated endpoints. Can you add the original one below for the moment, so it's there until the final deprecation on dec 17? Happy to use this as an opportunity to rename the fn too:

```
async getActiveOrderList(params) {
  return await this.request.get('v2/private/order/list', params);
}

/**
 * @deprecated use getActiveOrderList() instead
 */
async getActiveOrder(params) {
  return await this.request.get('open-api/order/list', params);
}
```